src/modules/hmis/hmisUtil.ts

```diff
@@ -371,6 +397,18 @@ export const serviceTypeSummary = (st: ServiceTypeFieldsFragment) => {
   return [st.category, st.name].join(': ');
 };
 
+export const customDataElementValueAsString = (
+  cde: CustomDataElementFieldsFragment
+): string | undefined => {
+  if (cde.value) return customDataElementValue(cde.value, 'for_display');
```

Comment:
This is fine, differentiating behavior on cde.value feels a bit fragile though. Logically we would use cde.repeats but that wouldn't make TS happy. Could handle this is with a Union type of CustomDataElement | CustomDataElementRepeated but that seems like a lot of refactoring for little gain.